### PR TITLE
Adding Collabs

### PR DESCRIPTION
**Pull request by BenWoodford76**
Adding Collaborators

### CODE REVIEW

File: _shows/17_18/collaborators.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- role: Technical Operator
+  name: Ben Woodford
+- role: Technical Operator
+  name: Nathan Penney
+- role: Technical Operator
+  name: Maddy Strauss
+- role: Technical Operator
+  name: James Curling
+- role: Technical Operator
+  name: Georgi Paxton
```
Comment (samozzy) [inline _shows/17_18/collaborators.md]: Had to cancel, your call as to whether to remain.

Comment (nhyty) [reply _shows/17_18/collaborators.md]: Would remove personally

**Comment by BenWoodford76** [comment]
Will drop Berrington a message about the individual cast posters

**Comment by BenWoodford76** [comment]
Am obtaining cast posters atm, will upload and include before merging

File: _shows/17_18/collaborators.md [hunk 2]
```diff
@@ -101,7 +99,30 @@ crew:
 assets:
   - type: poster
     image: 56TWtKJ
-
+  - type: poster
+    image: KbtWgLT
+  - type: poster
+    image:
```
Comment (nhyty) [inline _shows/17_18/collaborators.md]: why is this empty and here???

**Pull request merged**